xarray/tests/test_array_api.py

```diff
@@ -122,3 +122,45 @@ def test_where() -> None:
     actual = xr.where(xp_arr, 1, 0)
     assert isinstance(actual.data, Array)
     assert_equal(actual, expected)
+
+
+def test_statistics(arrays) -> None:
+    with xr.set_options(use_bottleneck=False):
```

Comment:
Haven't tried to make this work with bottleneck / numbagg yet, so turning off bottleneck temporarily for simplicity.